lib/src/main/java/org/team1540/rooster/drive/pipeline/FeedForwardProcessor.java

```diff
@@ -0,0 +1,74 @@
+package org.team1540.rooster.drive.pipeline;
+
+import java.util.OptionalDouble;
+import org.jetbrains.annotations.Contract;
+import org.jetbrains.annotations.NotNull;
+
+/**
+ * {@link Processor} to apply an <a href="https://www.chiefdelphi.com/media/papers/3402">Oblarg-style</a>
+ * feed-forward. This processor allows for velocity, acceleration, and static friction (vIntercept)
+ * feed-forwards. For further details, see {@link #apply(TankDriveData) apply()}. The output will
+ * mirror the input with the exception of a change in the feed-forward value.
+ */
+public class FeedForwardProcessor implements Processor<TankDriveData, TankDriveData> {
+
+  private final double kV;
+  private final double vIntercept;
+  private final double kA;
+
+  /**
+   * Creates a {@code FeedForwardProcessor} with the provided \(k_v\) and \(v_{Intercept}\)
+   *
+   * @param kV The velocity constant feed-forward, in output units per speed unit.
+   * @param vIntercept The velocity intercept, in output units
+   * @param kA The acceleration constant feed-forward, in output units per acceleration unit.
+   */
+  public FeedForwardProcessor(double kV, double vIntercept, double kA) {
+    this.kV = kV;
+    this.vIntercept = vIntercept;
+    this.kA = kA;
+  }
+
+  @Contract(pure = true)
+  private double getThrottle(double wantedSpeed, double wantedAccel) {
+    return (kV * wantedSpeed)
```

Comment:
in english goddamnit

Reply:
in french

Reply:
in morse code

Reply:
go read the link in the javadoc, but I will provide some more doc